src/gcinfo/gcinfoencoder.cpp

```diff
@@ -2661,7 +2661,7 @@ int BitStreamWriter::SizeofVarLengthUnsigned( size_t n, UINT32 base)
     // If a value gets so big we are probably doing something wrong
     _ASSERTE(((INT32)(UINT32)n) >= 0);
     _ASSERTE((base > 0) && (base < BITS_PER_SIZE_T));
-    size_t numEncodings = 1 << base;
+    size_t numEncodings = size_t{ 1 } << base;
```

Comment:
The previous line checks that `base` is lower then `BITS_PER_SIZE_T`, on `TARGET_64` it would be 64.
So then if `32 <  base < 64` it would produce the wrong result in the old variant.